sound/soc/codecs/rt715-sdca.c

```diff
@@ -300,7 +300,9 @@ static int rt715_sdca_put_volsw(struct snd_kcontrol *kcontrol,
 	.info = rt715_sdca_fu_info, \
 	.get = rt715_sdca_get_volsw, \
 	.put = rt715_sdca_put_volsw, \
-	.private_value = RT715_SDCA_FU_VALUE(reg_left, reg_right, xshift, \
+	//.private_value = RT715_SDCA_FU_VALUE(reg_left, reg_right, xshift, \
+	//				    xmax, xinvert)}
+	.private_value = SOC_DOUBLE_R_VALUE(reg_left, reg_right, xshift, \
```

Comment:
the whole hunk doesn't belong to this PR, right?

Reply:
> the whole hunk doesn't belong to this PR, right?

@lyakh Thanks. Done